addons/account/models/account.py

```diff
@@ -396,6 +396,7 @@ class AccountJournal(models.Model):
     _name = "account.journal"
     _description = "Journal"
     _order = 'sequence, type, code'
+    _inherit = ['mail.alias.activation.mixin']
```

Comment:
mail.alias.activation.mixin logic should be added to mail.alias.mixin
We will apply mail.alias.mixin on account.journal. Do you want to do this in this task?

Reply:
@Xavier-Do ok we will move mail.alias.activation.mixin to mail.alias.mixin. For account.journal the alias is used for purchase journal only. So inheriting mail.alias.mixin in account.journal will create useless alias for other journals also. Do you want me to do this change?

Reply:
@Xavier-Do changes has been done. Thanks!